src-opam/dockerfile_distro.ml

```diff
@@ -157,15 +159,16 @@ let win10_release_status v : win10_release_status = match v with
   | `V1909 -> `Deprecated
   | `V2004
   | `V20H2
-  | `V21H1 -> `Active
+  | `V21H1 | `Ltsc2022 -> `Active
 
-let win10_latest_release = `V21H1
+let win10_latest_release = `Ltsc2022
```

Comment:
This is the latest release of Windows 11! It's all a bit odd, given the "last" release of Windows 10 clearly isn't, but I think this can stay at `` `V21H1`` (especially given that we don't use it IIRC?)

Reply:
Changed back to `` `V21H1 ``.